src/EnsembleKalmanProcess.jl

```diff
@@ -423,6 +448,14 @@ function get_scheduler(ekp::EnsembleKalmanProcess)
     return ekp.scheduler
 end
 
+"""
+    get_accelerator(ekp::EnsembleKalmanProcess)
+Return accelerator type of EnsembleKalmanProcess.
+"""
+function get_accelerator(ekp::EnsembleKalmanProcess)
```

Comment:
It is good this was picked up by codecov
I think if you try to test it you will get an error that it doesnt exist because I don't think we export it,

You should go to the top of the file and add `export get_accelerator` and write a short test that calls it